tock/tock/middleware.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.contrib import auth
+from datetime import datetime, timedelta
+
+
+class AutoLogout():
```

Comment:
Well done!  Nice and succinct. :-)

However, this does appear to be constructed with the [older style of middleware classes](https://docs.djangoproject.com/en/1.11/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware), and I wonder if it might be worth getting this in shape with the [current middleware style](https://docs.djangoproject.com/en/1.11/topics/http/middleware/#writing-your-own-middleware) to help prep with a Django 2.0 update (technically this can be done now for 1.11, too).

This may or may not make for more work though given the other pieces of middleware, so there's an argument for not doing that right now, too.